var/ramble/repos/builtin/applications/namd/application.py

```diff
@@ -0,0 +1,326 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import re
+import os
+from ramble.appkit import *
+from ramble.expander import Expander
+from ramble.keywords import keywords
+
+
+class Namd(SpackApplication):
+    '''Define NAMD application'''
+    name = 'namd'
+
+    tags = ['molecular-dynamics']
+
+    default_compiler('gcc12', spack_spec='gcc@12.2.0', compiler_spec='gcc@12.2.0')
+
+    software_spec('impi2021p8', spack_spec='intel-oneapi-mpi@2021.8.0', compiler='gcc12')
```

Comment:
Another +1 here being able to take an app / ramble version to minimum spack version as this will fail on some slightly old but still fairly recent versions of spack

Reply:
Yeah, this is a bigger design decision that we need to make, but I'll make a task for it.